### PR TITLE
Pass isChecked in order to properly calculate 3rd state of checkbox

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

**Comment by mmenestr** [comment]
Do you mind adding a screenshot of it :D

File: packages/components/src/Components/BulkSelect/BulkSelect.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ visible unless you update it.');
                                                 aria-label="Select all"
                                                 onChange={ onSelect }
                                                 checked={ checked }
```
Comment (PanSpagetka) [inline packages/components/src/Components/BulkSelect/BulkSelect.js:51]: Do we need both `checked` and `isChecked` ?

**Comment by karelhala** [comment]
@mmenestr sure thing!

Screenshot from 2020-05-06 12-04-00

File: packages/components/src/Components/BulkSelect/BulkSelect.js (hunk 1 above)
Comment (karelhala) [reply packages/components/src/Components/BulkSelect/BulkSelect.js]: Yeah, I wanted to keep the checked for consistency reasons, but it works same way without it.

**Comment by mmenestr** [comment]
Looks good to me! Thanks, Karel!

**Pull request merged**